### DISCUSSION TITLE
My business account was rejected on "Employment verification" step

### DISCUSSION THREAD

**Discussion by xuelongjiang** [microsoft/MicrosoftEdge-Extensions#173] [Q&A]
![937a702d3687a9dda67d8a6ccc2e6bc](https://github.com/user-attachments/assets/14d4f87d-0cca-4652-b7c1-7a49a537a00d)
I registered a company type account, but during the verification process, I have been consistently rejected in the 'Employment verification' step! There are no specific prompts for errors, and there is no way to contact the backend staff!

**Comment by ManikanthMSFT** [comment]
Hi @xuelongjiang, could you please provide your seller ID for verification?

**Comment by xuelongjiang** [comment]
My seller id is 88525090, TKS!





---Original---
From: ***@***.***&gt;
Date: Fri, Jul 19, 2024 17:23 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [microsoft/MicrosoftEdge-Extensions] My business account wasrejected on "Employment verification" step (Discussion #173)




 
Hi @xuelongjiang, could you please provide your seller ID for verification?

**Comment by rajatrocks** [comment]
I am also experiencing the same thing. I just created an account, pressed the button in the verification email, and am similarly seeing a red X on "Employment verification". My seller id is 88543410.

thanks, - rajat

**Comment by ManikanthMSFT** [comment]
Hey @xuelongjiang and @rajatrocks, thank you for reaching out. We understand your eagerness for an update. Please be assured that our team is diligently working on the issue.

Once we have more information or an update on the timeline for resolution, we will promptly write back to you. Your understanding and patience are appreciated.

**Comment by rajatrocks** [comment]
Now whenever I press the Fix Now button, the page reloads and shows a clock on the "Verification started" step. If I wait a minute and reload, then I see it back to a red x on Employment Verification. I have uploaded an EIN doc from the government in the past, but now can't even see that.

**Comment by rajatrocks** [comment]
Hi there, I've been stuck here for a week. How can I get approved to add my extension to the Edge store? 

thanks, - rajat

**Comment by liambolling** [comment]
I'm having this issue too. Can you check my support request? 2407260010005759

**Comment by ManikanthMSFT** [comment]
Hi @xuelongjiang & @rajatrocks, we have emailed you the details required for the verification process of your extension. Please review the information and respond at your earliest convenience.

**Comment by rajatrocks** [comment]
I have sent in the requested doc, thanks Manikanth!

**Comment by liambolling** [comment]
Can you help me out too @ManikanthMSFT? Can you also explain why this isn't working for all of us?

**Comment by ManikanthMSFT** [comment]
Hi @liambolling, could you please contact us at ext_dev_support@microsoft.com so that we can assist you better.

**Comment by rajatrocks** [comment]
I submitted the requested info 2 weeks ago and still no change. What can I do to help move things along?
